.github/workflows/test.yml

```diff
@@ -17,13 +17,10 @@ jobs:
       fail-fast: false
       matrix:
         php_version:
-          - '7.2'
           - '7.4'
         dependencies:
           - 'default'
-        include:
-          - php_version: '7.2'
-            dependencies: 'lowest'
```

Comment:
Should keep this matrix job but just change the php_version to 7.4 (we still want to test with the composer lowest matching deps)

Reply:
Ahh yeh, oops. Sorted now